modules/dcache-nfs/src/main/java/org/dcache/chimera/nfsv41/door/NFSv41Door.java

```diff
@@ -858,6 +858,34 @@ public String call() throws IOException {
         }
     }
 
+
+    @Command(name = "show transfers filtered", hint = "show active transfers filtered",
```

Comment:
I'm wondering whether it would be better to update the existing command rather than adding a new command: the new command, when run without any filters, provides the same output as the `show transfers` command currently.